superset-frontend/package.json

```diff
@@ -92,7 +92,6 @@
     "@superset-ui/plugin-chart-echarts": "^0.16.4",
     "@superset-ui/plugin-chart-table": "^0.16.4",
     "@superset-ui/plugin-chart-word-cloud": "^0.16.4",
-    "@superset-ui/plugin-filter-antd": "^0.16.4",
```

Comment:
we moved it to the current repository, so better take from here

Reply:
I don't follow - That's why I removed it

Reply:
ah got it not correct, it's ok